frontend/src/lib/forms/OrgRoleSelect.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+  import { OrgRole } from '$lib/gql/types';
+  import t from '$lib/i18n';
+  import Select from './Select.svelte';
+
+  export let id = 'role';
+  export let value: OrgRole;
+  export let disabled = false;
+  export let verbose = false;
+  export let withLabel = false;
+  export let error: string | string[] | undefined = undefined;
+</script>
+
+{#if withLabel}
+<Select {id} bind:value label={$t('org_role.label')} {error}>
+  <option value={OrgRole.User}>
+    {$t(verbose ? 'org_role.user_description' : 'org_role.user')}
+  </option>
+  <option value={OrgRole.Admin}>
+    {$t(verbose ? 'org_role.admin_description' : 'org_role.admin')}
+  </option>
+</Select>
+{:else}
+<select {disabled} bind:value {id} class="select select-bordered" on:change>
```

Comment:
on change is only supported when not using the label, not sure that makes much sense. Can we make it consistent?